Examples/SwiftSyntaxBuilder.md

```diff
@@ -0,0 +1,30 @@
+# Swift Syntax Builder Examples
```

Comment:
Same here as for SwiftSyntax.md. I’d include the non-code parts as comments in the .swift file.